src/ArticlePage.jsx

```diff
@@ -18,20 +19,53 @@ export default function ArticlePage() {
       getArticleById(id).then((article) => {
         setIndividualArticle(article);
       });
+      getCommentsByArticleId(id).then((articleComments) => {
+        console.log(articleComments, "<<<");
+        setComments(articleComments);
+      });
     }
-  }, []);
+  }, [id]);
+  console.log(comments);
 
   return (
-    <section className="article">
-      <img
-        src={individualArticle.article_img_url}
-      />
-      <div className="body-content">
-        <h1>{individualArticle.title}</h1>
-        <h2>By {individualArticle.author}</h2>
-        <b>{individualArticle.body}</b>
-        <h3>{individualArticle.created_at ? formatDate(individualArticle.created_at) : ''}</h3>
-        <button>Upvote</button>
+    <section className="article-body">
+      <section className="article">
+        <img src={individualArticle.article_img_url} />
+        <div className="body-content">
+          <h1>{individualArticle.title}</h1>
+          <h2>By {individualArticle.author}</h2>
+          <b>{individualArticle.body}</b>
+          <h3>
+            {individualArticle.created_at
+              ? formatDate(individualArticle.created_at)
+              : ""}
+          </h3>
+          <button>Upvote</button>
+        </div>
+      </section>
+
+      <div className="comments-body">
+        <h2>Comments</h2>
+        <button>Filter</button>
+        <ul>
+          {comments.map((comment) => {
+            return (
+              <li key={comment.comment_id}>
+                <h3>{comment.author}</h3>
+                <p>
+                  {comment.created_at
+                    ? formatDate(individualArticle.created_at)
+                    : ""}
+                </p>
+                <div className="single-comment-body">
+                  <b>{comment.body}</b>
+                </div>
+                <b>{comment.votes}</b>
+                <button>Upvote</button>
+              </li>
+            );
+          })}
+        </ul>
```

Comment:
This looks good but it's getting to be quite a lot of `HTML` within this one component. You've now got some HTML for the individual article , and some for a list ( `<ul>` ) and some for the items inside the list... I would portion this out into some more components for better organisation

Reply:
for example 
`<CommentList />`
`<CommentCard />`

are two new components that could take some of this code